ansible/roles/ci/deploy/upload_debs/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Defaults aptly url
+
+aptly_url: http://10.10.10.62:8080
+repository_name: shadow-private
+gpg_password: shadow
```

Comment:
this password is probably ok to share in clear: the aptly server's ports can be closed to the outside, we just need to open the :80 port for http to serve the packages